pyVmomi/VmomiSupport.py

```diff
@@ -273,6 +274,106 @@ def __call__(self, **kwargs):
       else:
          raise AttributeError("'%s' does not exist" % self.name)
 
+
+class VmomiJSONEncoder(json.JSONEncoder):
+    """
```

Comment:
Indentation in pyVmomi code is 3 space. Looks like you did 4 space indentation.
Please can you check and change that?